sdk/eventhub/eventhubs-checkpointstore-tables/review/eventhubs-checkpointstore-tables.api.md

```diff
@@ -0,0 +1,25 @@
+## API Report File for "@azure/eventhubs-checkpointstore-tables"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { Checkpoint } from '@azure/event-hubs';
+import { CheckpointStore } from '@azure/event-hubs';
+import { PartitionOwnership } from '@azure/event-hubs';
+
+// @public
+export const logger: import("@azure/logger").AzureLogger;
+
+// @public (undocumented)
```

Comment:
Can you add documentation to this class? You can probably borrow heavily from the BlobCheckpointStore.

Reply:
Can you rerun `npm run build` for this package? This file should have been updated when you added docs for `TableCheckpointStore`.